### PR TITLE
default -g

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#28476

## Description
Setting the lint command to run by default on the changed packs only if no other argument is specified.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -374,7 +374,8 @@ def secrets(config, **kwargs):
                          "Package in docker image checks -  pylint, pytest, powershell - test, powershell - analyze.\n "
                          "Meant to be used with integrations/scripts that use the folder (package) structure. "
                          "Will lookup up what docker image to use and will setup the dev dependencies and "
-                         "file in the target folder. ")
+                         "file in the target folder. If no additional flags specifying the packages are given,"
```
Comment (reutshal) [inline demisto_sdk/__main__.py:377]:
```suggestion
                         "file in the target folder. If no additional flags specifying the packs are given,"
```

**Review by reutshal — APPROVED** [review]

**Comment by reutshal** [comment]
@roysagi Please take a look

**Review by reutshal — COMMENTED** [review]
@Noy-Maimon Please add this change to the RN

**Review by roysagi — COMMENTED** [review]
@Noy-Maimon - Please add a changelog entry.

**Comment by reutshal** [comment]
@roysagi Please review the changelog entry

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 * Added the *--skip-id-set-creation* flag to **validate** command in order to add the capability to run validate command without creating id_set validation.
 * Fixed an issue where **validate** command checked docker image tag on ApiModules pack.
 * Fixed an issue where **find-dependencies** did not calculate dashboards and reports dependencies.
+* Fixed the default run of **lint** to be only on changed files, if no additional flags specifying the packs are given.
```
Comment (roysagi) [inline CHANGELOG.md:11]:
```suggestion
* Running **demisto-sdk lint** will automatically run on changed files (same behavior as the -g flag).
```

**Comment by roysagi** [comment]
@Noy-Maimon - commit my changelog suggestion and then merge from master and rebuild.

**Pull request merged**